engine/class_modules/sc_shaman.cpp

```diff
@@ -7024,6 +7022,12 @@ void shaman_t::init_base_stats()
   if ( specialization() == SHAMAN_ELEMENTAL || specialization() == SHAMAN_ENHANCEMENT )
     resources.base[ RESOURCE_MAELSTROM ] = 100;
 
+  if ( specialization() == SHAMAN_RESTORATION )
+  {
+    resources.base[ RESOURCE_MANA ]                  = 20000;
+    resources.initial_multiplier[ RESOURCE_MANA ]    = 5;
```

Comment:
`spec.restorarion_shaman->effect(5)` has the multiplier. I don't know where one can find the base mana value.

Reply:
could just divide the base it has by default (100k, not sure where it gets that from) by that multiplier as well; just need the base mana to be 20k since the spells calculate their cost off of that. that way it would at least scale